packages/backend-common/src/service/lib/ServiceBuilderImpl.test.ts

```diff
@@ -33,4 +34,27 @@ describe('ServiceBuilderImpl', () => {
       expect(result!['upgrade-insecure-requests']).toBeUndefined();
     });
   });
+
+  describe('setCustomErrorHandler', () => {
+    it('adds custom error handler', () => {
+      const serviceBuilder = new ServiceBuilderImpl(module);
+      const customErrorHandler = (
+        error: Error,
+        req: Request,
+        res: Response,
+        next: NextFunction,
+      ) => {
+        console.log(req, res);
```

Comment:
I know it's test code but I'm not certain this logging is needed in here either.

Reply:
It's for typescript to prevent unused variable error, but I can change it or just add "// @ts-ignore"

Reply:
It's for typescript to prevent unused variable error, but I can change it or just add "// @ts-ignore"

Reply:
Ah, you should be able to rename `req` and `res` to `_req` and `_res` to make TS happy in this case then.

Reply:
Thanks for that hint! I've just made changes.